pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -249,6 +253,24 @@ func (p *PollerWorker) Poll(execAgentCommands agentCommandsExecutor, closeChan c
 	}
 }
 
+// Poll is the call to the guest agent using libvirt API.
+func pollApiOperation(execApiOperation apiOperationsExecutor, closeChan chan struct{}, callTick time.Duration) {
```

Comment:
The logic in this function should be the same as in the above `Poll()` method, so the behavior is not changed by this PR.
Initially, the polling interval is small, and then gradually increased, until a specified value is reached. This is useful, to check frequently at the start of the VM, when cloud-init may change parameters.

For example, the `sysApiCallTick` is 2 minutes, but it would start from 10 seconds and gradually increase to 2 minutes.

Reply:
Understood, thanks :)